src/Resolution.ts

```diff
@@ -217,7 +217,10 @@ export default class Resolution {
    */
   async ipfsHash(domain: string): Promise<string> {
     domain = this.prepareDomain(domain);
-    return await this.getNamingMethodOrThrow(domain).ipfsHash(domain);
+    return await this.getNamingMethodOrThrow(domain).record(
+      domain,
+      'ipfs.html.value',
+    );
```

Comment:
lets have a `private getRecord(domain, name)` that would call all 3 methods together.